.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation Issue
+about: 'Something in our doc is missing or incorrect '
+title: ''
+labels: Documentation
+assignees: ''
+
+---
+
+**Describe the Issue**
+A clear and concise what the issue
+
+**Feature Area**
+What's incorrect? What's missing?
```

Comment:
Maybe provide an example of a feature area (i.e. input system, how to guides, profile system)